packages/ws-worker/src/events/run-start.ts

```diff
@@ -31,7 +31,7 @@ export default async function onRunStart(
   };
 
   const versions = {
-    worker: pkg.version,
+    worker: loadVersions().engine,
```

Comment:
Ooh this isn't right - the worker and engine versions are different

Reply:
I had to rename it to version. It actually takes the version from package.json of the current scope